### PR TITLE
Fix kotlin native tests

### PR DESCRIPTION
**Pull request by raniejade**
//cc @charleskorn

### CODE REVIEW

File: integration-test/src/nativeMain/kotlin/Dummy.kt [hunk 1]
```diff
@@ -1,2 +1,2 @@
 // This is a workaround for https://github.com/JetBrains/kotlin-native/issues/2617
-object Dummy
+object Dummy
```
Comment (raniejade) [inline integration-test/src/nativeMain/kotlin/Dummy.kt:2]: issue close now, but it didn't make it on the `1.3.70` release. So either in `1.3.71` or `1.4.0`.

**Comment by charleskorn** [comment]
Nice - are you still planning to include this in an upcoming release?

**Comment by raniejade** [comment]
Yeah, `2.1` most likely. Going to make a feature branch for it. I'm planning to reuse the compiler extension you wrote for the JS implementation and maybe jvm as well.

**Comment by charleskorn** [comment]
Awesome, looking forward to it :)

**Pull request merged**